app/pages/search/controls/_search-controls.scss

```diff
@@ -10,6 +10,15 @@
   &__content {
     padding: 30px 0;
   }
+  &__municipalityRow {
+    height: 62px;
+  }
+  &__filter-dropdown-row {
+    height: 77px;
+  }
+  &__radio-buttons-row {
+    height: 105px;
```

Comment:
This is not a fix, those random value are h@cks, you should find better way to handle the root of problem, not just trying to make it work